assignments/arrays.js

```diff
@@ -63,32 +63,150 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+function findMe() {
+
+    let make;
+    let model;
+    let year;
+
+    for (let i = 0; i < inventory.length; i++) {
+
+        if (inventory[i].id === 33) {
+            make = inventory[i].car_make;
+            model = inventory[i].car_model;
+            year = inventory[i].car_year;
+            break;
+        }
+    }
+    return `Car 33 is a ${year} ${make} ${model}`;
+}
+
+console.log(findMe());
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+function lastCar() {
+
+    let lastCar = inventory.length - 1;
+    make = inventory[lastCar].car_make;
+    model = inventory[lastCar].car_model;
+    year = inventory[lastCar].car_year;
+
+
+    return `${year} ${make} ${model}`;
+
+}
+
+console.log(lastCar());
+
+
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
+function orderedList() {
+
+    let carModels = [];
+    let model;
+
+    for (let i = 0; i < inventory.length; i++) {
+
+        model = inventory[i].car_model;
```

Comment:
Good job on using variables for complicated values. It makes your code more clean and readable. 💯